templates/cf-jobs.yml

```diff
@@ -694,6 +694,7 @@ properties:
 
   router:
     enable_ssl: (( merge || nil ))
+    enable_routing_api: (( merge || true ))
```

Comment:
Given `router.enable_routing_api` defaults to `true` in the job spec, can we avoid having this default value also in the manifest template?